static/app/utils/fields/index.ts

```diff
@@ -432,16 +439,41 @@ export const SPAN_OP_FIELDS: Record<SpanOpBreakdown, FieldDefinition> = {
   },
 };
 
+export const REPLAY_FIELDS: Record<ReplayFieldKey, FieldDefinition> = {
+  [ReplayFieldKey.CountErrors]: {
+    desc: t('Number of errors in the replay'),
+    kind: FieldKind.FUNCTION,
+    valueType: FieldValueType.INTEGER,
+  },
+  [ReplayFieldKey.CountSegments]: {
+    desc: t('Number of segments in the replay'),
+    kind: FieldKind.FUNCTION,
+    valueType: FieldValueType.INTEGER,
+  },
+  [ReplayFieldKey.Duration]: {
+    desc: t('Duration of the replay'),
+    kind: FieldKind.FUNCTION,
+    valueType: FieldValueType.DURATION,
+  },
+  [ReplayFieldKey.ProjectId]: {
+    desc: t('Project ID'),
+    kind: FieldKind.FUNCTION,
+    valueType: FieldValueType.STRING,
+  },
+};
```

Comment:
Looks like these can show up up in the Issues typeahead. I think it's because an error has a tag called 'duration' so we're pulling that description in. It's happening in the sentry org, not the test one.

The error i see with the duration tag is ae8e2a014c58452cb2c53f23e467e8fd

<img width="1079" alt="Screen Shot 2022-09-09 at 1 31 19 PM" src="https://user-images.githubusercontent.com/187460/189438183-8693e882-8ddd-4fef-ad63-cf0c363aa4e6.png">

Reply:
@ryan953 I'm not sure how I can add an item for the duration field and have it not show up in that view that is in this screenshot. Issue and performance searches have a list of defined fields and duration isn't one of them so for me it doesn't show up anywhere except the replay search. Currently I just added field items for the other fields that didn't have descriptions and left duration out. Maybe theres a search input for "My Pinned Search" where I can omit duration or I guess I would need to make changes to the underlying `SmartSearchBar` component

Reply:
It kinda feels like we need two versions of `static/app/utils/fields/index.ts`, and then inject the correct data into the searchBar instead of importing it. Otherwise with this big list of field names, we're going to have `duration: "The Duration of the Replay in ms"` as a key that'll match with anyones custom tags. The smartBar needs to only look at keys & descriptions which are related to it's domain, not everything under the sun.